engine/util/src/main/scala/pl/touk/nussknacker/engine/util/json/BestEffortJsonEncoder.scala

```diff
@@ -32,7 +37,10 @@ case class BestEffortJsonEncoder(failOnUnkown: Boolean, highPriority: PartialFun
       case a: Double => safeDouble(a)
       case a: Int => safeInt(a)
       case a: Number => safeNumber(a.doubleValue())
+      case a: Boolean => safeJson[Boolean](fromBoolean) (a)
       case a: LocalDateTime => Encoder[LocalDateTime].apply(a)
+      //Default implementation serializes to ISO_ZONED_DATE_TIME which is not handled well by some parsers...
+      case a: ZonedDateTime => time.encodeZonedDateTimeWithFormatter(DateTimeFormatter.ISO_OFFSET_DATE_TIME).apply(a)
```

Comment:
Can't see handlers for `Instant` and `OffsetDateTime`. Will they be handled correctly?

Reply:
I added handling them